app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,54 @@
+
+
+
+  <h1>
+    ¿Qué tengo que hacer?
+  </h1>
+
+
+
+<div>
+<section >
+
+  <ul>
+    <il>
+      <%= link_to "Añadir Tarea", new_task_path %>
+    </il>
+    <il>
+      <%= link_to "Todas las tareas", tasks_path %>
+    </il>
+  </ul>
+</section>
+
+<section id="list">
+<h2>Lista de Tareas</h2>
+<ul>
+  <% @tasks.each do |task|  %>
+    <li>
+      <h3>
+        <% if task.complete %>
```

Comment:
This `if` statement can probably be DRY'd up somewhat.

The `link_to` that we're creating in both cases is identical except for the ID that it will have. Since that is the case we can probably use a ternary statement to figure out the correct ID and then write the `link_to` code once:
```erb
<%= link_to task.title, task_path(task.id), id: task.complete ? "Complete" : "NoComplete" %>
```